api/src/main/java/com/github/retrooper/packetevents/util/PEVersion.java

```diff
@@ -178,8 +180,8 @@ public boolean snapshot() {
 
     /**
      * Gets the snapshot commit hash. May be of any length.
-     * May always be present or absent, doesn't matter
-     * wether this is a snapshot version or not.
```

Comment:
Typo: "wether".
Claiming that the value is present or absent sounds pretty redundant, so this was already rephrased.